src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -186,23 +208,29 @@ void positiveAnonymiseProsecutorEntity() {
         assertLastModifiedByAndAt(prosecutorEntity, userAccount);
     }
 
-    @Test
-    void positiveAnonymiseTranscriptionCommentEntity() {
+    @ParameterizedTest(name = "Anonymise TranscriptionCommentEntity with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void anonymiseTranscriptionCommentEntity(boolean isManuallyRequested) {
         setupOffsetDateTime();
         TranscriptionCommentEntity transcriptionCommentEntity = new TranscriptionCommentEntity();
         transcriptionCommentEntity.setComment("comment");
 
         UserAccountEntity userAccount = new UserAccountEntity();
-        dataAnonymisationService.anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity);
+        doNothing().when(dataAnonymisationService).registerDataAnonymisation(any(), any(TranscriptionCommentEntity.class), anyBoolean());
+
+        dataAnonymisationService.anonymiseTranscriptionCommentEntity(userAccount, transcriptionCommentEntity, isManuallyRequested);
+
         assertThat(transcriptionCommentEntity.getComment()).matches(TestUtils.UUID_REGEX);
         assertThat(transcriptionCommentEntity.getLastModifiedBy()).isEqualTo(userAccount);
         assertThat(transcriptionCommentEntity.isDataAnonymised()).isTrue();
         assertLastModifiedByAndAt(transcriptionCommentEntity, userAccount);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, transcriptionCommentEntity, isManuallyRequested);
     }
 
 
-    @Test
-    void assertPositiveAnonymiseCourtCaseEntity() {
+    @ParameterizedTest(name = "Anonymise CourtCase with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void assertPositiveAnonymiseCourtCaseEntity(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated